src/main/java/se/sundsvall/invoicesender/integration/messaging/MessagingIntegration.java

```diff
@@ -55,9 +56,9 @@ public ItemStatus sendInvoice(final String path, final ItemEntity invoice, final
 		}
 	}
 
-	public void sendStatusReport(final List<BatchEntity> batches, final String municipalityId) {
+	public void sendStatusReport(final List<BatchEntity> batches, final LocalDate date, final String municipalityId) {
```

Comment:
Completely optional, but instead of passing a date around, you _could_ use the date from any of the batch entities.